### ISSUE TITLE
conditional batch normalization

### ISSUE THREAD

**Issue by lvyilan23** [jerryli27/TwinGAN#34]
Hi, could you please tell the difference of **instance_norm.py** and **batch_norm.py**?
If I only want to use conditional batch normalization , can I directly call the function **conditional_batch_norm()**  in **batch_norm.py** to process the feature map.
